src/scenes/Event/scenes/topic.js

```diff
@@ -370,23 +422,36 @@ export default class TopicPage extends React.Component {
       type: TransactionType.Withdraw,
       status: TransactionStatus.Success,
       topicAddress: address,
-      senderAddress: lastUsedAddress,
+      senderAddress,
     });
     if (pendingTxs.length > 0) {
       return {
+        show: true,
         disabled: true,
         message: <FormattedMessage
           id="withdrawDetail.alreadyWithdrawn"
           defaultMessage="You have already withdrawn with this address."
         />,
-        warningTypeClass: EventWarningType.Info,
+        warningTypeClass: classes.withdrawn,
+      };
+    }
+
+    // Can withdraw winning
+    const winniningAddresses = _.filter(walletAddressesWithWinnings, {
```

Comment:
can use `_.find()` for better efficiency. filter will iterate through the entire list (o(n) time), while find will stop at the first element found.